tests/src/nodes/nomos.rs

```diff
@@ -8,12 +8,17 @@ use consensus_engine::overlay::{FlatOverlaySettings, RoundRobin};
 use consensus_engine::NodeId;
 use nomos_consensus::{CarnotInfo, CarnotSettings};
 use nomos_http::backends::axum::AxumBackendSettings;
+#[cfg(feature = "libp2p")]
+use nomos_libp2p::Multiaddr;
+use nomos_libp2p::SwarmConfig;
```

Comment:
This probably needs to be in one line for the feature flag.
`use nomos_libp2p::{Multiaddr, SwarmConfig};`